src/packages/core/content-type/types.ts

```diff
@@ -1,9 +1,18 @@
 import type {
-	PropertyTypeContainerModelBaseModel,
 	CompositionTypeModel,
 	PropertyTypeModelBaseModel,
+	ReferenceByIdModel,
 } from '@umbraco-cms/backoffice/backend-api';
 
+export type UmbPropertyContainerTypes = 'Group' | 'Tab';
```

Comment:
@nielslyngsoe shouldn't this be in singular form? `UmbPropertyContainerType`
It seems to reflect a single type `Group` or `Tab`.

Like we have `CompositionTypeModel`, `PropertyTypeModelBaseModel`, `PropertyTypeModelBaseModel` etc. which are not called "models".